### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a)

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1164]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxmatrixdescription.mxp:nth-child(4) > a
#### Snippet
`<a href="/blog/2019/06/11/introducing-matrix-1-0-and-the-matrix-org-foundation">out of beta</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.25 (foreground color: #0098d4, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
